src/__tests__/__snapshots__/create-action-creator.dts.spec.ts.snap

```diff
@@ -1,35 +1,120 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`createActionCreator createActionCreator(
+exports[`createActionCreator const todoAdd = createActionCreator(
       '[Todo] add',
       // tslint:disable-next-line:no-inferrable-types
       resolve => (name: string, completed: boolean = false) =>
         resolve({ name, completed })
-    ) (type) should match snapshot 1`] = `"((name: string, completed?: boolean) => { type: \\"[Todo] add\\"; payload: { name: string; completed: boolean; }; }) & { type: \\"[Todo] add\\"; toString(): \\"[Todo] add\\"; }"`;
+    ) (type) should match snapshot 1`] = `"any"`;
```

Comment:
This is not true. I think you should snapshot like follow above and similar cases:

```ts
const a = createActionCreator('A')

// @dts-jest:pass:snap
a
```
```

Reply:
so let's create each action creator twice. One for snapshot and another for testing created action creators.

Reply:
as you wish.